app/views/unattended/provisioning_templates/snippet/preseed_networking_setup.erb

```diff
@@ -74,25 +70,4 @@ iface $real inet6 static
 <% end -%>
 <% end -%>
 EOF
-<% else -%>
-<%# Ubuntu 18.04+ uses /etc/netplan -%>
-cat << EOF >> /etc/netplan/01-netcfg.yaml
-    # <%= interface.identifier %>
-    $real:
-  <% if interface_dhcp -%>
-        dhcp4: true
-  <% else -%>
-      addresses:
-      <% addresses.each do |address| -%>
-        - <%= address %>
-      <% end -%>
-        <%if @host.ip && host_subnet && !host_dhcp && host_subnet.gateway -%>gateway4: <%= host_subnet.gateway %><% end %>
-        <%if @host.ip6 && host_subnet6 && !host_dhcp6 && host_subnet6.gateway -%>gateway6: <%= host_subnet6.gateway %><% end %>
-          nameservers:
-            <% if @host.domain %>search: [ <%= @host.domain %> ] <% end %>
-            addresses: [ <%= interface_subnet.dns_primary %><% if interface_subnet.dns_secondary -%>, <%= interface_subnet.dns_secondary %><% end -%> ]
-        dhcp4: false
-  <% end -%>
-EOF
-<% end -%>
 <% end -%>
```

Comment:
This is the commit no 1 which we should backport. The other changes are only to improve tests (do not backport).